en/docs/observability/root-cause-analysis.md

```diff
@@ -0,0 +1,99 @@
+3. Click on one request shown on the low code diagram (latencies). You can observe the status code for that request and thereby identify the exact API invocation that caused any error(s).
+
+### Diagnostics View
+The Diagnostics view provides the capability to view errors, throughput, latency, CPU, memory, and logs simultaneously for a particular event. You can detect and analyze errors and anomalies in detail via the diagnostics view.
+
+![Diagnostic View](/assets/img/observability/diagnostics-view.png)
+
+We refer to a horizontal section of the graph for a particular period as the **bin**. A bin consists of the following.
+
+- **Start time and end time**- Denoted by the dotted lines separating each bin.
+- **Logs**- Any logs that appeared during the bin's time range are listed on the logs column, along with the timestamp the logs started to appear and the log count for each log. The logs are listed according to precedence. The error logs will be listed first, followed by the info logs. Each bin shows a maximum of five log entries sorted based on the log count.
+- **Error**- Number of HTTP errors occurred at the selected time.
+- **TP**- Throughput of the requests at the selected time (req/s).  
+- **Latency**- Latency of the request at the selected time (ms).
+- **CPU**- CPU usage at the selected time (milicores).
+- **Memory**- Memory usage at the selected time (MiB)
+
+Let's see how to find the root cause by analyzing this data.
+
+Follow the steps below.
+
+1. Click on the **Diagnostics View** tab on the left panel. 
+
+    !!!info
+        The time range selected for the **Throughput & Latency** graphs will be applied to the **Diagnostics View** by default. We recommend capturing the time range you want to analyze in detail from the **Throughput & Latency** graph and then navigate to the **Diagnostics View**.
+        
+2. In the sample data, you can observe a couple of prominent errors appearing in the four bins.
+3. In the second bin during the period 01/04/2021 13:28:59 to 01/04/2021 13:41:59, you can observe 2420 occurrences of the error log "error while connecting to the hr-service”, along with other errors which occurred at a lesser frequency.
+
+    ![Diagnostic View - second bin(/assets/img/observability/second-bin.png/)
+](/assets/img/observability/second-bin.png/)
+
+4. Furthermore, when you look at the graphs at 17/02/2021 13:41:00, you can find a peak with 2221 error. At this exact timestamp, notice an increase in the throughput and a drop in latency. 
+5. By analyzing the graphs as above, you can confirm that the cause for the latency drop and the throughput spike is the connectivity issue displayed by the error logs.
+6. Observe that the error graph is also fluctuating over time as the service logs errors.
+7.  You can observe similar behavior in the third and fourth bins as well.
+
+   ![Diagnostic View - other bin](/assets/img/observability/other-bins.png)
+   
+   
```

Comment:
Extra new lines?

Reply:
Intentionally added for UX alignment purposes. The extra lines won't be visible when the page renders.